### ISSUE TITLE
Link to estimote-sdk.aar is broken

### ISSUE THREAD

**Issue by lawjeremy** [Estimote/Android-SDK#246]
I would like to download the estimote aar file, but the link in the readme is broken

**Comment by wafel82** [comment]
Hi, 
I'm not sure which link you are referring to. So, I'll try to clarify things a little bit :)
Our SDK is distributed by the JCenter repository. Thats why you have to ensure you have such repo configured in your project. Usually, it's done by the following lines in top-level build.gradle of your project:
```gradle
allprojects {
    repositories {
        jcenter()
        (all other repositories you are using goes here)
    }
}
```
Once you have JCenter configured, add this line to your build.gradle file:
```gradle
dependencies {
  compile 'com.estimote:sdk:1.0.12'
}
```
Hope this address your confusions  :)

Regards
Wojtek

**Comment by lawjeremy** [comment]
Sorry, I suppose I should have been more clear.

In the manual installation instructions: 

[https://github.com/Estimote/Android-SDK/blob/master/Docs/manual_installation.md](
https://github.com/Estimote/Android-SDK/blob/master/Docs/manual_installation.md) 

It says to download estimote-sdk.aar, but the link is broken. I would like to download the aar to include in an Appcelerator project I am working on.

Thanks.

**Comment by wafel82** [comment]
Hi,

Got it. the manual_installation instruction you are referring to is definitely out-of-date and should be fixed.
Latest version of the Estimote SDK for Android can be downloaded from this link: 
[https://jcenter.bintray.com/com/estimote/sdk/1.0.12/sdk-1.0.12.aar](https://jcenter.bintray.com/com/estimote/sdk/1.0.12/sdk-1.0.12.aar)
You can refer to the enclosing folders structure to get the bigger picture of the stuff we are delivering: [https://jcenter.bintray.com/com/estimote/](https://jcenter.bintray.com/com/estimote/)

**Please note that the official project structure we are supporting is the regular Java/Kotlin gradle/maven-based setup.**
One important thing you have to keep in mind when you are not using gradle/maven is that the libraries we are delivering are **NOT** "fat-jars" anymore. 
In other words - All internal or third-party libraries our SDK depends on is not directly included "inside" our SDK library file. All stuff our SDK depends on is simply listed out as a maven/gradle dependency. 
I'm not familiar with Appcelerator build tooling and how it handles libraries with external dependencies (which is super-common case in java world :) ). However - if you simply add just indoorsdk-1.0.0.aar to your build and you end up with nasty "class def not found" exception, then you'll have to add all libraries we are depending on manually.

Regards
Wojtek

**Comment by lawjeremy** [comment]
That's great, thanks! I'm actually experimenting at the moment to work out problems with the build process before attempting to use this in a real project.
